src/ui/app-icon.svelte

```diff
@@ -9,23 +9,23 @@
     <img class="rcb-app-icon" {src} alt="App icon" />
   </picture>
 {:else}
-  <div class="rcb-app-icon loading" />
+  <div class="rcb-app-icon loading"></div>
 {/if}
 
 <style>
-  .rcb-app-icon {
-    width: 40px;
-    height: 40px;
-    border-radius: 12px;
-    box-shadow: 0px 1px 10px 0px rgba(0, 0, 0, 0.1);
-    margin-right: 16px;
-  }
+    .rcb-app-icon {
+        width: 40px;
+        height: 40px;
+        border-radius: 12px;
+        box-shadow: 0px 1px 10px 0px rgba(0, 0, 0, 0.1);
+        margin-right: 16px;
+    }
 
-  .rcb-app-icon-picture-container {
-    height: 40px;
-  }
+    .rcb-app-icon-picture-container {
+        height: 40px;
+    }
 
-  .rcb-app-icon.loading {
-    background-color: gray;
-  }
+    .rcb-app-icon.loading {
+        background-color: gray;
+    }
```

Comment:
these also aren't formatted correctly

Reply:
Please check the formatting PR here:
https://github.com/RevenueCat/purchases-js/pull/251